install/crd/config.authorino.3scale.net_services.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.1
+    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
You might need to update your controller-gen. Upgraded by @eguzki in #143.

Reply:
I will rebase and force push when we are both happy with the PR (so we don't lose any comments).

Do you mind having a lot of commits from development or should I squash it?

Reply:
I usually try to keep a clean history. It doesn't have to be linear, but without "broken" commits as much as possible.